learning_journal/development.ini

```diff
@@ -0,0 +1,81 @@
+###
+# app configuration
+# http://docs.pylonsproject.org/projects/pyramid/en/1.7-branch/narr/environment.html
+###
+
+[app:main]
+use = egg:learning_journal
+
+pyramid.reload_templates = true
+pyramid.debug_authorization = false
+pyramid.debug_notfound = false
+pyramid.debug_routematch = false
+pyramid.default_locale_name = en
+pyramid.includes =
+    pyramid_debugtoolbar
+
+# sqlalchemy.url = postgres://VMB@localhost:5432/learning_journal_db
+
+sqlalchemy.url = postgres://ikdxlhvnqxfaba:AApK0JBTnIaH-nCLHzPTZVK-C3@ec2-54-225-112-215.compute-1.amazonaws.com:5432/d4gog6q4qtk508
```

Comment:
You should absolutely **not** be storing this value in your `.ini` file in github.  It has no protection on it beyond what is here.  I can now use this database directly from my machine.  Please uninstall the postgres addon you have installed to delete this database, and then re-add it to get a new one.  Use your OS environment to store these values not git.